CONTRIBUTING.md

```diff
@@ -1,5 +1,18 @@
-Contributing to the CloudML Samples
-----------------------------------
+# Contributing to Cloud ML Engine Samples
 
-Thanks for the enthusiasm, but currently we are not accepting external
-contributions!
+You want to contribute to Cloud ML Engine samples? That's awesome! Please refer to the short guide below.
+
+# Contributing Guide
+
+We are happy to see your enthusiasm. This table lists the criteria and how important is it to follow when you make your contributions. 
+
+<br/>
+
+| Criteria        | Core         | Contrib  |
+| ------------- |:-------------:| -----:|
+| Include requirements.txt for dependencies     | Hard | Hard |
+| Include unit tests     | Hard | Soft |
+| Include integration tests     | Hard | Soft|
+| Include core TF API aka no contrib   | Hard | Soft |
```

Comment:
I don't undersatnd this one. Can you explain this a bit more?

Reply:
The idea is to list which criteria is hard (required) and which is soft (optional). Does that change in wording help or the way it reads its unclear what this table really is?

Reply:
Also not clear exactly what Core or Contrib refers to, after a few minutes I figured out it refers to the samples themselves but my first thought was it referred to TF core or TF contrib.

Reply:
@elmer-garduno @rhaertel80 how about now?